### PR TITLE
Miscellaneous improvements and removals

### PR DESCRIPTION
**Pull request by triphora**
Got kind of carried away here when trying to do MOD-215, if there's anything you want me to revert don't hesitate to ask :sweat_smile:

Alternative commit title: Project Perses Part Πρώτος

### CODE REVIEW

File: src/routes/v1/versions.rs [hunk 1]
```diff
@@ -242,44 +233,6 @@ pub async fn get_version_from_hash(
...
-
-    if let Some(id) = result {
-        Ok(HttpResponse::TemporaryRedirect()
-            .append_header(("Location", &*id.url))
-            .json(DownloadRedirect { url: id.url }))
-    } else {
-        Ok(HttpResponse::NotFound().body(""))
-    }
-}
-
```
Comment (Geometrically) [inline src/routes/v1/versions.rs:282]: this shouldn't be removed

**Review by Geometrically — CHANGES_REQUESTED** [review]
lgtm besides one change

File: src/routes/v1/versions.rs (hunk 1 above)
Comment (triphora) [reply src/routes/v1/versions.rs:282]: I thought that was a private route, but oki

Comment (triphora) [reply src/routes/v1/versions.rs:282]: Actually this is identical to the equivalent v2 route so I can keep it deleted and just call that

Comment (Geometrically) [reply src/routes/v1/versions.rs:282]: Not sure if it was private but iirc people did use it

File: src/routes/health.rs [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-            "ready": false,
-            "reason": "Indexing is not finished"
-        });
-        return HttpResponse::InternalServerError().json(data);
-    }
-    HttpResponse::Ok().json(json!({
-        "ready": true,
-        "reason": "Everything is OK"
-    }))
-}
```
Comment (Geometrically) [inline src/routes/health.rs:31]: Also should keep this, it's useful in the future

**Review by Geometrically — CHANGES_REQUESTED** [review]

**Review by Geometrically — APPROVED** [review]

**Pull request merged**